env.faucet.config.json

```diff
@@ -58,6 +58,60 @@
         "masked": true,
         "type": "string"
       }
+    },
+    "BOT": {
+      "BACKEND_URL": {
+        "description": "full url for the bot to reach the backend",
+        "default": "http://localhost:5555",
+        "type": "string"
+      },
+      "DRIP_AMOUNT": {
+        "description": "default amount of token to send",
+        "default": 0.5,
+        "type": "number"
+      },
+      "MATRIX_ACCESS_TOKEN": {
+        "description": "your bot access token here is how to find it https://t2bot.io/docs/access_tokens/.",
+        "required": true,
+        "masked": true,
+        "type": "string"
+      },
+      "MATRIX_BOT_USER_ID": {
+        "description": "your bot user id",
+        "required": true,
+        "regexp": "^@.*:.*$",
+        "type": "string"
+      },
+      "MATRIX_SERVER": {
+        "description": "matrix server url",
+        "default": "https://matrix.org",
+        "type": "string"
+      },
+      "NETWORK_DECIMALS": {
+        "description": "decimal amount for the network",
+        "default": 12,
+        "type": "number"
+      },
+      "NETWORK_UNIT": {
+        "description": "token unit for the network",
+        "default": "UNIT",
+        "type": "string"
+      },
+      "FAUCET_IGNORE_LIST": {
+        "description": "A list of Matrix accounts that will be silently (but logged) ignored, comma separated. Example: \"@alice:matrix.org,@bob:domain.com\"",
+        "default": "",
+        "type": "string"
+      },
+      "DEPLOYED_REF": {
```

Comment:
i think these configs ideally should be merged together under SMF root, as still we're duplicating DEPLOYED_REF and potentially other configs. While the deployment is the same, the configs could be de-duplicated as the source anyway the same

Reply:
100% agree that this should be de-duplicated.

However I left it duplicated in order to limit changes in a single PR.
Having no changes to env vars, there is less to review and less to think about when deploying it.

I was planning to approach those env vars separately if/when this PR goes through.

Reply:
I'm ok with leaving this change for a second PR, if you have already contemplated these factors